app/views/patients/show.html.erb

```diff
@@ -92,15 +92,37 @@
           <%passes_threshold = assessment.symptom_passes_threshold(column)%>
           <%symptom_value = assessment.get_reported_symptom_value(column)%>
           <% if assessment.get_reported_symptom_by_name(column)&.type == "BoolSymptom"%>
-          <% if symptom_value == true %>
-          <% symptom_value = "Yes" %>
-          <% elsif symptom_value == false %>
-          <% symptom_value = "No"%>
-          <%end%>
-          <td><% if passes_threshold==true -%><span class="concern"><%=symptom_value%></span><% elsif passes_threshold==false -%><%=symptom_value%><%else -%><% end -%></td>
+            <% if symptom_value %>
+              <% symptom_value = "Yes" %>
+            <% else symptom_value %>
+              <% symptom_value = "No"%>
+            <%end%>
+            <td>
+              <% if passes_threshold.nil? -%>
+                &nbsp;
+              <% elsif passes_threshold -%>
+                <span class="concern"><%=symptom_value%></span>
+              <% else -%>
+                <%=symptom_value%>
+              <% end -%>
+            </td>
           <% end %>
           <% if assessment.get_reported_symptom_by_name(column)&.type == "FloatSymptom" || assessment.get_reported_symptom_by_name(column)&.type == "IntegerSymptom"%>
-          <td><% if passes_threshold==true -%><span class="concern"><%=symptom_value%></span><% elsif passes_threshold==false -%><%=symptom_value%><%else -%><% end -%></td>
+            <td>
+              <!-- Reported symptomatic via nonspecific means such as voice or sms -->
+              <% if passes_threshold.nil? -%>
+                &nbsp;
+              <!-- Reported asymptomatic via nonspecific means such as voice or sms -->
+              <% elsif symptom_value == 100_000 || symptom_value == 100_000.0 || symptom_value == 0 || symptom_value == 0.0 -%>
```

Comment:
Using 0 as a reserved value for "Unknown" concerns me, as 0 is a realistic value that could be actually reported

Reply:
Do you have a number in mind

Reply:
-100,000?